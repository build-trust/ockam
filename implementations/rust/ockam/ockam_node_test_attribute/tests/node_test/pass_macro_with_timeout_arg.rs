#[ockam_node_test_attribute::node_test(timeout = 1000)]
async fn my_test(ctx: &mut Context) -> ockam::Result<()> {
    ctx.stop().await.unwrap();
}
