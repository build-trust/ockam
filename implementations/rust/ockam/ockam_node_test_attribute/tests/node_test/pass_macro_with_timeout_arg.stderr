error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/pass_macro_with_timeout_arg.rs:1:1
  |
1 | / #[ockam_node_test_attribute::node_test(timeout = 1000)]
2 | | async fn my_test(ctx: &mut Context) -> ockam::Result<()> {
3 | |     ctx.stop().await.unwrap();
4 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/node_test/pass_macro_with_timeout_arg.rs`
